static/js/admin/src/IngestFromSourceForm.js

```diff
@@ -139,10 +154,12 @@ define('IngestFromSourceForm', [ 'jquery', 'AbstractFileUploadForm', 'ModalLoadi
 			$this.parents(".file_browse_heading").first().nextUntil(".file_browse_heading", ".file_browse_entry").each(function() {
 				if (select) {
 					$(this).addClass("selected");
+					$("input[name=ingest]").prop("checked", select);
 				} else {
 					$(this).removeClass("selected");
+					$("input").prop("checked", select);
```

Comment:
Won't this selector find all the input elements on the page now?

Reply:
No, it just finds all the checkboxes in the selected row

Reply:
maybe i'm misunderstanding, but i thought jquery would use the whole document as the context for a lookup if you don't specify a context. Is it somehow inheriting a context?